server/bin/pbench-tarball-stats.py

```diff
@@ -108,7 +131,9 @@ def transform_buckets(buckets: dict, sat_names: list):
         for name, total in sorted(satellites.items()):
             pct = (total / count) * 100.0
             sats_pct[name] = pct
-        new_buckets[bucket_name] = dict(count=count, sats_pct=sats_pct)
+        new_buckets[bucket_name] = dict(
+            count=count, sats_pct=sats_pct, partial=(now_s == bucket_name)
```

Comment:
I suggest adding a comma to the end of this line.  That will cause Black to format it with each pair on its own line, which I personally would prefer.

Reply:
I added the comma and then ran it through `black` but nothing changed.

Reply:
And with or without the comma at the end, `black` forced it back into this form.

Reply:
That's...interesting.  IIRC, the added comma is supposed to keep it from being unwrapped back to a single line; perhaps, it only matters if you manually wrap it, first.

Oh, well, nevermind....